### ISSUE TITLE
Are you planning the snippets for the new sintax?!

### ISSUE THREAD

**Issue by carlosvillu** [justinj/vim-react-snippets#3]

**Comment by justinj** [comment]
Hey, I was talking about this with @mileszs at #2, I don't know if he's still interested in making a separate repo or not. @mileszs what are you thinking?

**Comment by kevinold** [comment]
I'm likely going to fork this and add updated snippets soon. 



On July 4, 2015 at 8:12:06 PM CDT, Justin Jaffray <notifications@github.com> wrote:Hey, I was talking about this with @mileszs at #2, I don't know if he's still interested in making a separate repo or not. @mileszs what are you thinking?  —Reply to this email directly or view it on GitHub.

**Comment by carlosvillu** [comment]
I dont think so that we need a new repo for this. Maybe a PR is just enough.

If I have time these days I will do it my self

**Comment by mileszs** [comment]
I apologize for not getting back to you yet on [my fork and pull request](https://github.com/justinj/vim-react-snippets/pull/2), @justinj. Now that I think a bit harder about it, though, that is probably a good indication that I should not try maintaining another open-source project. :-)

Perhaps the community can just help to maintain an es6 branch on this repo?

**Comment by letientai299** [comment]
[The official tool to generate React app](https://github.com/facebookincubator/create-react-app) use ES6 class syntax. So I think we should update the snippets soon. 

I have started a fork that support for es6. Unfortunately, I only use UltiSnips, hence, only make change to the UltiSnips version. For that reason, I don't think my change would be accepted as a pull request. But it might help someone who interesting in React with ES6. 

Please review it here: https://github.com/letientai299/vim-react-snippets/tree/es6 
(I created new branch, master is untouched)

@justinj How do you think about linking my fork in the Readme?

**Comment by justinj** [comment]
I've been hesitant to update this one since I didn't want to break someone's editor config, but I guess ES6 is probably standard enough by this point that I'm causing more problems by keeping it this way than I'm preventing. I'm happy to update this to one of the (many) ES6 forks and direct people to a separate ES5 repo.

I'm also unconfident about maintaining both types of snippets (UltiSnips were added in a PR a while back) but if you would be willing to supply the UltiSnips version I can update the snipmate version.

Does that sound ok to you @letientai299?

**Comment by letientai299** [comment]
It's ok for me. I'm more than willing to supply my fork. Pull request is comming.
